docs/Coiote_IoT_DM/multi_firmware_update.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Multi-component Firmware Update
+date: 2023-06-05
+og_title: AVSystem IoT Developer Zone
+---
+
+# Multi-component Firmware Update
+
+Remotely update a component of your LwM2M device's firmware using the **Advanced Firmware Update Object** `/33629`.
+
+The Advanced Firmware Update Object supports multiple instances, each representing a "component" of the device's firmware that can be upgraded separately. The specific meaning and purpose of these components are not standardized and can vary depending on the implementation. However, they typically encompass elements such as bootloaders, application code, cellular modem firmwares, security processor firmwares, and other related firmware entities.
+
+## Prerequisites
+
+- An active [Coiote DM](https://eu.iot.avsystem.cloud/) user account
+- A device which supports the Advanced Firmware Update Object `/33629`
+    
+## Firmware Update Object `/33629`
+
+Object `/33629` defines the update process using **4 Update States** representing the phase of the update process. It knows a total of **13 Update Results**  representing the most common outcomes of the firmware update process.
```

Comment:
The notion that an object "knows" some update results sounds weird to me.

Maybe rephrase it as something like "Additionally, a total of **13 Update Results** may be reported, representing (...)"?